packages/cf-style-const/src/variables.js

```diff
@@ -14,6 +14,20 @@ export default {
     seven: 'linear-gradient(left, #F69259, #FFDB6F)',
     eight: 'linear-gradient(left, #85CBA8, #FFDB6F)'
   },
+  color: {
+    rain: '#408BC9',
```

Comment:
As long as each hex code is only defined once I don't really care.  For example:
```
color.blue =  '#408BC9';
color.rain = color.blue;
color.success = color.rain;
```